common/config/rush/common-versions.json

```diff
@@ -83,23 +83,29 @@
       "~0.6.1" // API Extractor is using an older version of source-map because newer versions are async
     ],
 
-    "webpack": ["~5.35.1"],
+    "webpack": ["~4.44.2"],
```

Comment:
Should this move down to the new `    // --- For Webpack 4 projects ----` section?

Reply:
fixed